### PR TITLE
View All Link for Project Summary List

### PR DESCRIPTION
**Pull request by cdcabrera**
View All for project summary listing, always enabled

### CODE REVIEW

**Comment by dtaylor113** [comment]
Screenshot would be really helpful :-)

File: src/components/projects-summary/projects-summary.html [hunk 1]
```diff
@@ -30,7 +30,7 @@ <h4 class="catalog-modal-title">
     </a>
   </div>
   <div ng-if="$ctrl.projects && $ctrl.projects.length" class="catalog-project-summary-list">
-    <div ng-if="$ctrl.totalProjects > $ctrl.maxDisplayProjects" class="projects-count">
+    <div class="projects-count">
       <strong>{{$ctrl.maxDisplayProjects}}</strong>
       of
       <strong>{{$ctrl.totalProjects}}</strong>
```
Comment (spadgett) [inline src/components/projects-summary/projects-summary.html]: @beanh66 Should we be showing "3 of 3 Projects" in addition to the view all link?

Comment (beanh66) [inline src/components/projects-summary/projects-summary.html]: @spadgett @cdcabrera Yes the project count should be displayed as well. As long as there is at least 1 project, the project count and view all link should be shown.

Comment (spadgett) [inline src/components/projects-summary/projects-summary.html]: Thanks, should it say "3 of 3" or just "3"?

Comment (beanh66) [inline src/components/projects-summary/projects-summary.html]: "3 of 3"  so the user is clear that they may not see anything additional in the "View All" 👍

**Pull request merged**